docs/integration-tests.md

```diff
@@ -0,0 +1,74 @@
+## Integration Tests
+
+Debugger.html integration test are end-to-end tests that simulate a user debugging an application.
```

Comment:
Yup. just wrote it down quickly for the cases where people *are* using the tool and might be confused